pennylane/templates/subroutines/__init__.py

```diff
@@ -35,3 +35,4 @@
 from .basis_rotation import BasisRotation
 from .qsvt import QSVT, qsvt
 from .select import Select
+from .ctrlsequence import CtrlSequence
```

Comment:
I think this is best in the "[Other subroutines](https://docs.pennylane.ai/en/stable/introduction/templates.html#other-subroutines)" category (though we probably need to break up that category).